streetcrm/admin.py

```diff
@@ -539,7 +539,6 @@ def advanced_search_do(self, request, form, export):
         # Remove any without participants
         # results = self._remove_empty_values(results)
 
-        
```

Comment:
Not a big deal, but it's good to avoid unrelated whitespace changes -- they are just a distraction in the diff, since the reviewer will assume that any visible textual change is part of the semantic change (and then discover otherwise).